### ISSUE TITLE
Cannot read property `toLowerCase()` of undefined: Preventing chrome extension from working entirely

### ISSUE THREAD

**Issue by Explosion-Scratch** [ghostery/ghostery-extension#735]
### Description

![image](https://user-images.githubusercontent.com/61319150/147841266-0820d98a-1bf2-44d8-abf9-7dce7f8eb31b.png)

### Expected Behavior

I expect an extension with millions of users to work

### Actual Behavior

An error shown in the screenshot above.

### Steps to Reproduce

1. Install the extension
2. Check errors

### Versions

+ Browser: Chrome
+ OS: ChromeOS
+ Node:
+ NPM:

### Notes:
I think it might be this:
https://github.com/ghostery/ghostery-extension/blob/f55ef022ac28dd0be3e8b97e4f7056c4128e5f03/src/classes/Globals.js#L150-L159

Probably caused because ChromeOS isn't able to be parsed or something like that. As an additional note my userAgent is:
```
'Mozilla/5.0 (X11; CrOS x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.0.0 Safari/537.36'
```

And here is the output of running `parser(navigator.userAgent)` using the `ua-parser-js` library:

```json
{
  "ua": "Mozilla/5.0 (X11; CrOS x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.0.0 Safari/537.36",
  "browser": {
    "name": "Chrome",
    "version": "96.0.0.0",
    "major": "96"
  },
  "engine": {
    "name": "Blink",
    "version": "96.0.0.0"
  },
  "os": {},
  "device": {},
  "cpu": {
    "architecture": "amd64"
  }
}
```

**Comment by Explosion-Scratch** [comment]
Made a fix in #736

**Comment by Explosion-Scratch** [comment]
Fixed in #736
